snapshots/lcow/lcow.go

```diff
@@ -368,8 +368,8 @@ func (s *snapshotter) createSnapshot(ctx context.Context, kind snapshots.Kind, k
 				}
 			} else {
 				var sizeGB int
-				if sizeGBstr, ok := snapshotInfo.Labels[rootfsSizeLabel]; ok {
-					i64, _ := strconv.ParseInt(sizeGBstr, 10, 32)
+				if sizeGBstr, ok := snapshotInfo.Labels[rootfsSizeInBytesLabel]; ok {
+					i64, _ := strconv.ParseInt(sizeGBstr, 10, 64)
```

Comment:
This doesn't work with a simple replace of the label, this is providing the size in bytes and where this ends up expects it in GiB's. You need a conversion somewhere in here https://github.com/containerd/containerd/blob/a570c8184aa3dc578f22a5200124e1d875659357/snapshots/lcow/lcow.go#L425-L464

Reply:
thanks for review.
rebased it.